frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/import-providers-tab-validation.ts

```diff
@@ -40,7 +40,8 @@ export const setImportProvidersTabValidity = (options: UpdateOptions) => {
       VMWizardTab.IMPORT_PROVIDERS,
       result.isValid,
       result.hasAllRequiredFilled,
-      result.error,
+      result.errorKey,
+      result.fieldKeys,
```

Comment:
this is not consistent - same things should be sent to the check as it is to the dispatch